pom.xml

```diff
@@ -70,8 +70,8 @@
       <module>server/integration</module>
       <module>as-modules/embedded</module>
       <module>as-modules/client</module>
+      <module>integrationtests</module>
       <module>integrationtests/as-integration-embedded</module>
-      <module>integrationtests/as-integration-client</module>
```

Comment:
Double checked - you are right. Nice eye, thank you for spotting that. Fix and rebase is on the way